nixos/modules/services/networking/xrdp.nix

```diff
@@ -91,88 +103,118 @@ in
       };
 
       confDir = mkOption {
-        type = types.path;
```

Comment:
Please keep the type here, it is important to ensure we are passing the correct types.

Reply:
Done